src/xunit.analyzers/TypeSymbolFactory.cs

```diff
@@ -11,5 +11,20 @@ internal static INamedTypeSymbol IEnumerableOfObjectArray(Compilation compilatio
             var iEnumerableOfObjectArray = iEnumerableOfT.Construct(objectArray);
```

Comment:
It took me a few seconds to work this out, so please confirm: you suggest I put in line 10 in this file the following:
```csharp
var objectArray = GetObjectArrayType(compilation);
```
right?